ThePensionsRegulator.Frontend/Styles/_tpr-button.scss

```diff
@@ -55,6 +49,7 @@ a.govuk-button--secondary:link, a.govuk-button--secondary:visited {
 .govuk-file-upload::file-selector-button:hover,
 .govuk-button--secondary:link:active,
 .govuk-button--secondary:visited:active,
+input[type="file"]::file-selector-button:hover,
```

Comment:
You shouldn't need `input[type="file"]::file-selector-button:hover` and `.govuk-file-upload::file-selector-button:hover`. 

If the component doesn't have `.govuk-file-upload` applied we shouldn't be styling it.

Also, you've moved all the other file upload styles to `_tpr-file-upload.scss` (which is good) but why not this?

Reply:
removed